fee_simulator/core/round_fee_distribution/leader_timeout_150_previous_normal_round.py

```
@@ -76,43 +79,61 @@ def apply_leader_timeout_150_previous_normal_round(
-                round_label="LEADER_TIMEOUT_150_PREVIOUS_NORMAL_ROUND",
-                role="VALIDATOR",
-                vote=normalize_vote(votes[addr]),
-                hash="0xdefault",
-                cost=0,
-                staked=0,
-                earned=budget.validatorsTimeout
-                + split_amount(appeal_bond, len(majority_addresses)),
-                slashed=0,
-                burned=0,
+    if majority == "UNDETERMINED":
+        for addr in votes.keys():
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)